### ISSUE TITLE
Latest ESX breakes everything

### ISSUE THREAD

**Issue by MonsterZockerHD** [jonassvensson4/jsfour-atm#28]
Hey, I really like the atm script but since the new ESX update we have some minor issues which prevent us from using it: 
1) the cash is not showing, just 0 (atm's and bank form)
2) the bank money is not showing, just 0 (atm's and bank form)
3) you can't withdraw or deposit money (atm's and bank form)
4) at the bank form you can't see your own first and lastname (if you want to deposit, withdraw or transfer money)
5) the NUI isn't showing up anymore at the banks

**Comment by jnsvns** [comment]
I'm not sure how I'm supposed to help you. I haven't been using ESX for months and I haven't tried out the newest version. This is also not just an ESX resource anymore, since I rewrote the resource I removed ESX as an dependency. Their functions however is still implemented and should run if you've es_extended started.

You'll have to debug it yourself, the only file that has anything to do with ESX is the [server.js](https://github.com/jonassvensson4/jsfour-atm/blob/master/server/server.js) so it shouldn't be a huge deal. 

I will try to help you the best I can.

1. [server.js#43-47](https://github.com/jonassvensson4/jsfour-atm/blob/master/server/server.js#L43-L47) Does it get inside this if statement? If so, double check that those ESX functions are still working. They might have changed those?
2. You said the bank NUI isn't showing up anymore. You're probably getting an error in the client console since something is preventing it from running. What does it say?
3. I've also had a look in the new SQL file that es_extended provides. Since they've removed essentialmode as an dependecy I can't see where they're storing the users first and lastname. Where is it stored?
